kustomize/preprod/simple-app/deployment.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: simple-app-deployment
   labels:
     app: simple-app
+  annotations:
+    reloader.stakater.com/auto: "true"
```

Comment:
This annotation is not necessary 

See https://github.com/stakater/Reloader?tab=readme-ov-file#notes

Secrets: https://github.com/stakater/Reloader?tab=readme-ov-file#notes
Configmap: https://github.com/stakater/Reloader?tab=readme-ov-file#notes

According to the 2 examples, we don't need the `auto` annotation